### PR TITLE
chore(docs): adding documentation for Keptn on Docusaurus (docs.tracetest.io)

### PR DESCRIPTION
**Pull request by danielbdias**
This PR aims to add a "Tools and Integrations" section to our documentation and add Keptn docs there.

To validate these docs locally, you can:

1. Open the terminal on repository root folder
2. Go to `./docs`
3. Run `npm install && npm start`
4. After installing the dependencies and compiling docusaurus, a browser window should open with the documentation.

## Changes

- Added section "Tools and Integrations" on our docs
- Added Keptn docs

## Fixes

- #1883 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/docs/tools-and-integrations/integrations/keptn.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
+# Running Tracetest with Keptn
```
Comment (danielbdias) [inline docs/docs/tools-and-integrations/integrations/keptn.md:1]: This documentation is the same as presented on https://github.com/kubeshop/tracetest/tree/main/examples/keptn-integration (I only needed to fix the links to the files).

After the contents of this documentation are published on docs.tracetest.io, I'll open another PR changing the docs there to reference this document.

File: docs/docs/tools-and-integrations/integrations/keptn.md [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+ 
+[Tracetest](https://tracetest.io/) is a testing tool based on OpenTelemetry that permits you to test your distributed application. It allows you to use your trace data generated on your OpenTelemetry tools to check and assert if your application has the desired behavior defined by your test definitions.
+
+[Keptn](https://keptn.sh/) is a powerful tool to automate the lifecycle of your application running on Kubernetes. One of the tasks that we can do on `keptn` is to test an application and see if it is healthy and ready to be used by your users.
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/integrations/keptn.md:5]:
```suggestion
[Keptn](https://keptn.sh/) is a powerful tool to automate the lifecycle of your application running on Kubernetes. One of the tasks that we can do with `keptn` is to test an application and see if it is healthy and ready to be used by your users.
```

File: docs/docs/tools-and-integrations/integrations/keptn.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+        - run
+        - --definition
+        - /keptn/data/test-definition.yaml
+        - --wait-for-result
+
+```
+
+## Quickstart
+
+Here we will show how to use Tracetest to do these tests and help in your delivery and testing workflows using the [Pokeshop](https://docs.tracetest.io/pokeshop/) example, available at address `http://demo-pokemon-api.demo` at Kubernetes.
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/integrations/keptn.md:36]:
```suggestion
Here we will show how to use Tracetest to run these tests and help in your delivery and testing workflows using the [Pokeshop](https://docs.tracetest.io/pokeshop/) example, available at `http://demo-pokemon-api.demo` in a Kubernetes cluster.
```

File: docs/docs/tools-and-integrations/integrations/keptn.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+
+### Prerequisites
+
+On your Kubernetes cluster you should have:
```
Comment (adnanrahic) [inline docs/docs/tools-and-integrations/integrations/keptn.md:40]:
```suggestion
In your Kubernetes cluster you should have:
```

**Review by adnanrahic — CHANGES_REQUESTED** [review]
Just some small edits needed. Really nice writing!

File: docs/docs/tools-and-integrations/integrations/keptn.md (hunk 3 above)
Comment (jfermi) [inline docs/docs/tools-and-integrations/integrations/keptn.md:36]:
```suggestion
Here we will show how to use Tracetest to do these tests and help in your delivery and testing workflows using the [Pokeshop](https://docs.tracetest.io/pokeshop/) example, available at `http://demo-pokemon-api.demo` for Kubernetes.
```

**Comment by adnanrahic** [comment]
Could you also edit this readme file to have a link to the docs?
https://github.com/kubeshop/tracetest/blob/main/examples/keptn-integration/README.md

If this readme file and the docs are identical, you can delete most of the readme file and briefly explain what the code does and link to the docusaurus docs.

File: docs/docs/tools-and-integrations/integrations/keptn.md [hunk 5]
```diff
@@ -0,0 +1,172 @@
...
+keptn add-resource --project keptn-tracetest-integration --service pokeshop --stage production --resource job-config.yaml --resourceUri job/config.yaml
+```
+ 
+### 3. Setup Job Executor Service to see events emitted by the test step.
+ 
+To guarantee that our job will be called by Keptn when we execute the `deployment` sequence, we need to configure the integration `Job Executor Service` on `keptn-tracetest-integration` project to listen to `sh.keptn.event.test-services.triggered` events. We can do that only through the Keptn Bridge (their Web UI), by going to our project, choosing the `Settings` option, and later `Integrations`.
+ 
+Choose the `job-executor-service` integration, and add a subscription to the event `sh.keptn.event.test-services.triggered` and the project `keptn-tracetest-integration`.
+ 
+### 4. Run sequence when needed
```
Comment (jfermi) [inline docs/docs/tools-and-integrations/integrations/keptn.md:165]:
```suggestion
### 4. Run sequence when needed.
```

File: docs/docs/tools-and-integrations/overview.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Overview
+
+Can be integrated and used with another tools. See below which integrations we are working on Tracetetest:
```
Comment (jfermi) [inline docs/docs/tools-and-integrations/overview.md:3]:
```suggestion
Tracetest can be integrated and used with another tools. See below which integrations we are working on:
```

File: docs/docs/tools-and-integrations/overview.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
...
+
+## Integrations
+
+- [Keptn](./integrations/keptn.md): Keptn is a powerful tool to automate the lifecycle of your application running on Kubernetes. With this integration, you can run Tracetest along Keptn test tasks to validate systems managed by Keptn.
```
Comment (jfermi) [inline docs/docs/tools-and-integrations/overview.md:7]:
```suggestion
- [Keptn](./integrations/keptn.md): Keptn is a powerful tool to automate the lifecycle of your application running on Kubernetes. With this integration, you can run Tracetest alongside Keptn test tasks to validate systems managed by Keptn.
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Review by jorgeepc — APPROVED** [review]

**Review by adnanrahic — APPROVED** [review]

**Pull request merged**